post.hbs

```diff
@@ -33,9 +33,29 @@ into the {body} of the default.hbs template --}}
             {{/if}}
 
             <section class="post-full-content">
-                <div class="post-content">
-                    {{content}}
-                </div>
+                {{#if @labs.members}}
+                    {{#if html}}
+                        <div class="post-content">
+                            {{content}}
+                        </div>
+                    {{else}}
+                        <div class="post-content"
+                            data-members='true'
+                            data-members-drop-in-resource="post"
```

Comment:
Nothing else uses "drop-in" so maybe `data-members-resource-type` might fit better?

Reply:
Yeah thats much better 👍 `drop-in` doesn't sound generic too

Reply:
Cool! @rishabhgrg I'm making this change, so can you update the SDK to work with it?